hudi-common/src/main/java/org/apache/hudi/avro/AvroSchemaUtils.java

```diff
@@ -109,6 +109,11 @@ public static Schema createNullableSchema(Schema.Type avroType) {
     return Schema.createUnion(Schema.create(Schema.Type.NULL), Schema.create(avroType));
```

Comment:
Let's rebase this one onto new one you're adding

Reply:
done.